elements/pfe-button/src/pfe-button.scss

```diff
@@ -0,0 +1,125 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: button;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(ui-accent),
+  BackgroundColor--hover: pfe-var(ui-accent--hover),
+  Color: pfe-var(ui-base--text),
+  FontSize: pfe-var(font-size),
+  Border: 0,
+  BorderRadius: pfe-var(surface--border-radius),
+  Padding: calc(#{pfe-var(container-padding)} / 2) pfe-var(container-padding),
+  LineHeight: pfe-var(line-height),
+  after: (
+    BorderColor: transparent,
+    BorderColor--hover: transparent,
+  ),
+);
+
+// Nested internal variables (pfe-local calls), maps cannot "self-reference"
+$LOCAL-VARIABLES: map-deep-merge(
+  $LOCAL-VARIABLES,
+  (
+    after: (
+      Border: pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-local(BorderColor, $region: after),
+      Border--hover: pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-local(BorderColor--hover, $region: after),
+    ),
+  )
+);
+
+:host {
+  display: inline-block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-variant="primary"]) button,
+button {
+  background-color: pfe-local(BackgroundColor);
+  color: pfe-local(Color);
+  font-size: pfe-local(FontSize);
+  font-family: pfe-var(font-family, $fallback: inherit);
+  padding: pfe-local(Padding);
+  cursor: pointer;
+  border-radius: pfe-local(BorderRadius);
+  border: pfe-local(Border);
+  position: relative;
+  line-height: pfe-local(LineHeight);
+
+  &::after {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    content: "";
+    border: pfe-local(Border, $region: after);
+    border-radius: pfe-local(BorderRadius);
+  }
+
+  &:hover,
+  &:focus {
+    --pfe-button--BackgroundColor: #{pfe-local(BackgroundColor--hover)};
+    --pfe-button__after--Border: #{pfe-local(Border--hover, $region: after)};
+  }
+}
+
+:host([pfe-variant="secondary"]) {
+  --pfe-button--BackgroundColor: transparent;
+  --pfe-button--BackgroundColor--hover: transparent;
+  --pfe-button--Color: #{pfe-var(ui-accent)};
+  --pfe-button__after--BorderColor: #{pfe-var(ui-accent)};
+  --pfe-button__after--Border--hover: 2px #{pfe-var(ui--border-style)};
+}
+
+:host([pfe-variant="tertiary"]) {
+  --pfe-button--BackgroundColor: transparent;
+  --pfe-button--BackgroundColor--hover: transparent;
+  --pfe-button--Color: #{pfe-var(text)};
+  --pfe-button__after--BorderColor: #{pfe-var(text)};
+  --pfe-button__after--Border--hover: 2px #{pfe-var(ui--border-style)};
+}
+
+:host([pfe-variant="danger"]) {
+  --pfe-button--BackgroundColor: #{$pf-color-red-100}; // feedback--critical?
+  --pfe-button--BackgroundColor--hover: #{$pf-color-red-200}; // feedback--critical--darkest is too dark
+}
+
+:host([pfe-variant="control"]) {
+  --pfe-button--BackgroundColor: transparent;
+  --pfe-button--BackgroundColor--hover: transparent;
+  --pfe-button--Color: #{pfe-var(text)};
+  --pfe-button--BorderRadius: 0;
+  --pfe-button__after--BorderColor: #{$pf-color-black-200}; // surface--border--lightest?
```

Comment:
@eyevana @kylebuch8 I think these question mark comments should likely be addressed